packages/styles/scss/patterns/sub-patterns/tableofcontents/_tableofcontents.scss

```diff
@@ -97,6 +97,14 @@
         a:hover {
           cursor: pointer;
         }
+
+        @supports (-moz-appearance: none) {
+          a:focus {
+            outline: none;
+            border: 1px solid $interactive-04;
+            box-shadow: 0 0 4px 0 $interactive-04;
+          }
+        }
```

Comment:
Did we reach out to design to see what the proper focus state is? I'm not seeing any focus state on here. Are we using the proper component? I say this because the problem may have already been solved with a different component, or we maybe able to solve it for all browsers instead of just Firefox.

Reply:
I'll reach out to them, this css is just a try of mimic the chrome focus state

Reply:
Right, but Carbon has its own focus state that we should probably be following.

Reply:
There was no interactive specs for TOC in the initial specs. I just spoke to Raph and we decided he would enable Carbon focus state using Carbon's `$focus` token so it's aligned with the system.

Reply:
@photodow I've updated the SCSS in order to be compliant with Carbon system.